src/parameters/cmp_0m.jl

```diff
@@ -0,0 +1,22 @@
+"""
+    CloudMicrophysicsParameters0M{FT}
+
+Parameters for zero-moment bulk microphysics scheme
+
+# Fields
+$(DocStringExtensions.FIELDS)
+"""
+struct CloudMicrophysicsParameters0M{FT} <: AbstractCloudMicrophysicsParameters
+    "precipitation timescale"
+    τ_precip::FT
+    "specific humidity precipitation threshold"
+    qc_0::FT
+    "supersaturation precipitation threshold"
+    S_0::FT
+end
+
+CloudMicrophysicsParameters0M(param_set) = CloudMicrophysicsParameters0M(
```

Comment:
I think we should construct this directly from the `.toml` parameters file, but I would prefer to do that in a different PR!